R/bmerge.R

```diff
@@ -105,10 +105,12 @@ bmerge = function(i, x, icols, xcols, roll, rollends, nomatch, mult, ops, verbos
           if (!is.null(attributes(i[[ic]]))) attributes(val) = attributes(i[[ic]])  # to retain Date for example; 3679
           set(i, j=ic, value=val)
           set(callersi, j=ic, value=val)       # change the shallow copy of i up in [.data.table to reflect in the result, too.
+        } else {
+          if (verbose) cat("Coercing integer column x.",names(x)[xc]," to type double to match type of i.",names(i)[ic]," which contains fractions.\n",sep="")
```

Comment:
Does `setNumericRounding` come into play here? If so we should note such (in the `verbose` message and/or in dox)

Reply:
Yes it comes into play.  But it does anyway for regular double to double type-matching joins too.  `setNumericRounding` is really just to resolve numeric accuracy; more like very-slight-inaccuracy.  I'm not sure the wording to use to best convey this. I just got swamped by another 11 revdep fails in the full-rerun so I'll try and get through those as priority.  If you have something in mind for the message here, feel free to add in a follow-up PR, or raise an issue to follow-up so we don't forget. Agree.